xbmc/music/MusicDatabase.h

```diff
@@ -140,6 +140,8 @@ class CMusicDatabase : public CDatabase
    \param rating [in] a rating for the song
    \param userrating [in] a userrating (my rating) for the song
    \param votes [in] a vote counter for the song rating
+   \param strSongvideoURL [in] url to video of the song
+   \param strSongVideoThumb [in] url to thumb of song video
```

Comment:
Questioning making the videothumb part of song CRUD. This thumb is just some type of art, and we have deliberately moved away from having fixed parameters for those. I understand it relates to the videoURL (which does belong here), but the prime relationship is some art for the song. So designwise where AddSong gets called it could be followed by an AddArt call rather than having the thumb (some special art) as a parameter of AddSong